src/dmd/attrib.d

```diff
@@ -1577,3 +1573,32 @@ int foreachUda(Dsymbol sym, Scope* sc, int delegate(Expression) dg)
         });
     });
 }
+
+/**
+ * Iterates the UDAs attached to the given symbol, without performing semantic
+ * analysis.
+ *
+ * Use this function instead of `foreachUda` if semantic analysis of `sym` is
+ * still in progress.
+ *
+ * If `dg` returns `!= 0`, it will stop the iteration and return that
```

Comment:
make this a regular `Returns:` ddoc section

Reply:
This is copied verbatim from the documentation of `foreachUda`, so the change will need to be made in both places.